### ISSUE TITLE
This application cannot tree shake icons fonts. It has non-constant instances of IconData

### ISSUE THREAD

**Issue by elirehema** [pd4d10/file-icon#1]
flutter build apk break with warning message 

> This application cannot tree shake icons fonts. It has non-constant instances of IconData at the following locations:
 > - file:///home/eli/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/file_icon-1.0.0/lib/file_icon.dart:34:7
 >                                                                      
 >                                                                      
>FAILURE: Build failed with an exception.

**Comment by pd4d10** [comment]
Yeah, because the code point is dynamic. `--no-tree-shake-icons` is required

**Comment by elirehema** [comment]
@pd4d10 my suggestion was you should include that information to  [README](https://pub.dev/packages/file_icon) i have struggled with CICD build [failures](https://github.com/ospic/mobile/runs/2257771179?check_suite_focus=true) about to drop the plugin then i found that solution in SO.

**Comment by pd4d10** [comment]
Thanks! It's a good idea.

Could you please update your PR to add this?

**Comment by elirehema** [comment]
@pd4d10 i have made PR

**Comment by pd4d10** [comment]
Unfortunately #2 would not work, because the file name is dynamic

The alternative way is to add this info in readme

**Comment by romanr** [comment]
why not add this in README? everybody stumble on the same error and waste time looking for solution
